mmv1/products/identityplatform/terraform.yaml

```diff
@@ -28,6 +28,16 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           billing_acct: :BILLING_ACCT
         # Resource creation race
         skip_vcr: true
+      - !ruby/object:Provider::Terraform::Examples
+        name: "identity_platform_config_monitoring"
+        primary_resource_id: "default"
+        vars:
+          instance_name: "memory-cache"
+        test_env_vars:
+          org_id: :ORG_ID
+          billing_acct: :BILLING_ACCT
+        # Resource creation race
```

Comment:
I copied this line from the example above, they seem to be similar: https://github.com/GoogleCloudPlatform/magic-modules/blob/f6c18a8db48c01f8f327df4a269df6e9bbb0a599/mmv1/products/identityplatform/terraform.yaml#L21-L40

Reply:
That's because that test has two instances of google_project_service. Even that should be fixable.

Can you remove skip_vcr and instead add the following to the google_identity_platform_config resource within that test:
depends_on = [google_project_service.identitytoolkit]

Reply:
Done, https://github.com/GoogleCloudPlatform/magic-modules/pull/7025/commits/762a4defaad78cbabf2b7c235974d650604c0ef6